Mathlib/Analysis/Normed/Field/ProperSpace.lean

```diff
@@ -0,0 +1,37 @@
+/-
+Copyright (c) 2024 Yakov Pechersky. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yakov Pechersky
+-/
+
+import Mathlib.Analysis.Normed.Field.Lemmas
+import Mathlib.Analysis.SpecificLimits.Basic
+import Mathlib.Topology.MetricSpace.ProperSpace
+
+/-!
+# Proper nontrivally normed fields
+
+This file contains a direct result of normed fields that are `ProperSpace`s.
+
+## Main results
+
+* `ProperSpace.of_weaklyLocallyCompactSpace_of_nontriviallyNormedField`
+-/
+
+open Metric Filter
+
+/-- A weakly locally compact normed field is proper. -/
+lemma ProperSpace.of_weaklyLocallyCompactSpace_of_nontriviallyNormedField
```

Comment:
Minor nitpick: your name has the instance arguments listed in the opposite order from how they appear.